system/CodeIgniter.php

```diff
@@ -780,11 +773,15 @@ protected function generateCacheName(Cache $config): string
     }
 
     /**
-     * Replaces the elapsed_time tag.
+     * Replaces the elapsed_time and memory_usage tag.
      */
     public function displayPerformanceMetrics(string $output): string
     {
-        return str_replace('{elapsed_time}', (string) $this->totalTime, $output);
+        return str_replace(
```

Comment:
This should probably be an after Filter at some point on the Welcome route - no business being in the main app class.

Reply:
Yes, I thought the same thing.
And probably Web Page Caching also should be a filter.